code/datums/spells/cluwne.dm

```diff
@@ -24,15 +24,16 @@
 	if(mind)
 		mind.assigned_role = "Cluwne"
 
-	var/obj/item/organ/internal/honktumor/cursed/tumor = new
-	tumor.insert(src)
+
 	dna.SetSEState(GLOB.nervousblock, 1, 1)
 	singlemutcheck(src, GLOB.nervousblock, MUTCHK_FORCED)
 	rename_character(real_name, "cluwne")
 
 	unEquip(w_uniform, 1)
 	unEquip(shoes, 1)
 	unEquip(gloves, 1)
+	var/obj/item/organ/internal/honktumor/cursed/tumor = new
+	tumor.insert(src)
```

Comment:
Is there some order of operations issue you're fixing by moving these lines down here?

Reply:
I imagine its because ```unEquip(shoes, 1)``` has to come before it, considering that clown shoes remove the waddle when they are unequipped.

Reply:
yeah the problem was adding the tumor then removing the shoes which borked the waddle signal